### DISCUSSION TITLE
Are different ports on the same domain land in different partitions?

### DISCUSSION THREAD

**Discussion by jcubic** [privacysandbox/privacy-sandbox-dev-support#231] [General] [answered]
I have a library for Cross-Domain communication and someone asked if he can use IP addresses with a different port (he uses a web app on the intranet), I'm not able to test this right now. So I'm asking here. Right now my library throws an error for this user because it's an untrusted context and I use `requestStorageAccess` to throw an error in that case.

So the question is does the same domain but a different port work the same as subdomains? Or they are different partitions and Cross-Domain communication will be blocked?

**Comment by johannhof** [comment, marked as answer]
Storage Partitioning and Storage Access generally operate on [sites](https://web.dev/articles/same-site-same-origin), which is ignoring ports. However, web storage and communication channels are often bound to origin (which includes the port). I'm not 100% sure which one you're asking for here, but it's definitely a good idea to test if you want to make sure.

**Reply by jcubic** [reply]
I'm asking about `Storage Partitioning and Storage Access`. This explains everything. I also get a message from the user of my library that the previous version of the code works for different ports. Which means it works the same as subdomains.

Thanks.

**Reply by johannhof** [reply]
Great, thanks for the update!
